test

```diff
@@ -17,7 +17,7 @@ GOSIMPLE_UNUSED_PATHS=$(go list ./... | sed -e 's/github.com\/coreos\/etcd\///g'
 # Invoke ./cover for HTML output
 COVER=${COVER:-"-cover"}
 
-GO_BUILD_FLAGS=-a
+GO_BUILD_FLAGS="-a -v"
```

Comment:
Do you see that as a problem?  Are they used for anything other than to see what failed?

With `-v` go will print the name of each translation unit, so it is easy to see which one failed.